ansible/group_vars/all.yml

```diff
@@ -581,6 +581,8 @@ monasca_control_plane_project: "monasca_control_plane"
 
 elasticsearch_address: "{{ kolla_internal_fqdn }}"
 enable_elasticsearch: "{{ 'yes' if enable_central_logging | bool or enable_freezer | bool or enable_osprofiler | bool or enable_skydive | bool or enable_monasca | bool else 'no' }}"
+enable_elasticsearch_curator: "no"
```

Comment:
yes if enable_elasticsearch?

Reply:
Actually, I see why it should be off by default.

But when we turn it on, I think it should do something by default.